Formula/gwt.rb

```diff
@@ -1,8 +1,8 @@
 class Gwt < Formula
   desc "Google web toolkit"
   homepage "http://www.gwtproject.org/"
-  url "https://storage.googleapis.com/gwt-releases/gwt-2.8.2.zip"
```

Comment:
In the project page, it still shows 2.8.2 though.

Reply:
cc @Bo98

Reply:
made a release notes update request, https://github.com/gwtproject/gwt-site/issues/298